applications/svZeroDGUI/static/js/app.js

```diff
@@ -60,22 +76,22 @@ function updateBoundaryConditionTypeVisibility() {
     cy.domNode(); // Register the domNode extension for making HTML classes for each node to assist Cypress testing
 
     cy.on('tap', function(event) {
-       if (event.target === cy) {
+       if (event.target === cy) {  // Filters clicks to those within the cytoscape drawing container.
             let nodeName = document.getElementById('node-name').value.trim();
             let nodeType = document.getElementById('node-type').value;
             let boundaryConditionType = nodeType === 'boundary_condition' ? document.getElementById('boundary-condition-type').value : null;
 
             if (nodeName !== "") {
                 let pos = event.position || event.cyRenderedPosition;
-//                let nodeId = node_count[nodeType];
                 let nodeId = `${nodeType}-${node_count[nodeType]}`;
                 node_count[nodeType] += 1;
 
                 let color;
                 if (nodeType === 'boundary_condition') {
+                    // Sets the icons and edge color for each node
                     switch (boundaryConditionType) {
                         case 'FLOW':
-                            color = '#FF00FF'; //hexadecimal
+                            color = '#FF00FF';
```

Comment:
Would be good to retain the comment saying what color the code corresponds to (for all the colors below)